packages/client/lib/execution/vmexecution.ts

```diff
@@ -31,8 +31,8 @@ export class VMExecution extends Execution {
   private pendingReceipts?: Map<string, TxReceipt[]>
   private vmPromise?: Promise<number>
 
-  /** Number of maximum blocks to run per iteration of {@link VMExecution.run} */
-  private NUM_BLOCKS_PER_ITERATION = 50
+  /** Maximally tolerated block time before giving a warning on CL */
+  private MAX_TOLERATED_BLOCK_TIME = 12
```

Comment:
Not sure, but maybe we want this as a CLI option as well?

Reply:
Ah, yes, that's a good idea, not sure if I'll add here, maybe along some other updates, but generally makes sense! 🙂